runtime/src/serde_snapshot/newer.rs

```diff
@@ -57,6 +57,7 @@ struct DeserializableVersionedBank {
     unused_accounts: UnusedAccounts,
     epoch_stakes: HashMap<Epoch, EpochStakes>,
     is_delta: bool,
+    prior_roots: Vec<Slot>,
```

Comment:
GitHub won't let me comment on the lines I actually want... Please remove `unused` and `unused_accounts` (and `UnusedAccounts`) (and from both Ser and Deser).